subprojects/gradle/cd/src/main/kotlin/com/avito/ci/BuildStepListExtension.kt

```diff
@@ -29,7 +29,7 @@ open class BuildStepListExtension(internal val name: String, objects: ObjectFact
     internal val steps: ListProperty<BuildStep> = objects.listProperty(BuildStep::class.java).empty()
 
     //todo property
```

Comment:
It also requires to migrate all other public build steps variables to properties for the sake of consistency.
I think it's out of scope for now and not a blocker.